### PR TITLE
Improve CGMES doc

### PR DESCRIPTION
**Pull request by miovd**
TODO
- detail export

**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Doc update

### CODE REVIEW

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 1]
```diff
@@ -404,10 +500,105 @@ This post-processor is enabled by adding the name `cgmesDLImport` to the list as
...
+#### Switch
+
+PowSyBl [`Switches`](../model/index.md#breakerswitch) are exported as CGMES `Breakers`, `Disconnectors` or `LoadBreakSwitches` depending on its `SwitchKind`.
+
+<span style="color: red">TODO details</span>
+
+#### ThreeWindingsTransformer
+
+PowSyBl [`ThreeWindingsTransformers`](../model/index.md#three-windings-transformer) are exported as CGMES `PowerTransformers` with three CGMES `PowerTransformerEnds`.
+
```
Comment (So-Fras) [inline pages/documentation/grid/formats/cim-cgmes.md:582]: Isn't there a need for details here too ?

```suggestion
<span style="color: red">TODO details</span>
```

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 2]
```diff
@@ -275,28 +272,127 @@ A CGMES `AsynchronousMachine` is mapped to a PowSyBl [`Load`](../model/index.md#
...
+If the CGMES `Switch` has its ends both inside the same voltage level, it is mapped to a PowSyBl [`Switch`](../model/index.md#breakerswitch) with attributes as described below:
+- `SwitchKind` is defined depending on the CGMES class
+  - If it is a CGMES `Breaker`, it is `BREAKER`
+  - If it is a CGMES `Disconnector`, it is `DISCONNECTOR`
+  - If it is a CGMES `LoadBreakSwitch`, it is `LOAD_BREAK_SWITCH`
+  - Otherwise, it is `BREAKER`
+- `Retained` is copied from CGMES `retained` if defined in node-breaker. Else, it is `false`.
+- `Open` is copied from CGMES SSH `open` if defined. Else, it is copied from CGMES `normalOpen`. If neither are defined, it is `false`.
+
+If the CGMES `Switch` has its ends in different voltage levels inside the same IGM, it is mapped to a PowSyBl [`Switch`] but the voltage levels, and potentially the substations, that contain its ends are merged: they are mapped to only one voltage level and/or substation.
```
Comment (So-Fras) [inline pages/documentation/grid/formats/cim-cgmes.md:388]: The link towards "switch" is missing.

```suggestion
If the CGMES `Switch` has its ends in different voltage levels inside the same IGM, it is mapped to a [`Switch`](../model/index.md#breakerswitch) but the voltage levels, and potentially the substations, that contain its ends are merged: they are mapped to only one voltage level and/or substation.
```

**Review by So-Fras — CHANGES_REQUESTED** [review]
Great documentation work, thank you.
I suggest 2 minor changes.

Is it also possible to take advantage of this PR and modify https://github.com/powsybl/powsybl.github.io/pull/287/files#diff-fd13120da0badc1b6c93c0ceb1e9caf17cc76b9b3e57b9adedf5e257febfadb1R461 in order to remove the reference to powsybl-diagramµ? The CGMES-related code has not been updated for a while in this repo.

File: pages/documentation/grid/formats/cim-cgmes.md [hunk 3]
```diff
@@ -275,28 +272,127 @@ A CGMES `AsynchronousMachine` is mapped to a PowSyBl [`Load`](../model/index.md#
...
 
 #### ExternalNetworkInjection
-<span style="color: red">TODO</span>
+
+CGMES `ExternalNetworkInjections` are injections representing the flows from an entire external network.
+
+A CGMES `ExternalNetworkinjection` is mapped to a PowSyBl [`Generator`](../model/index.md#generator) with attributes created as described below:
+- `MinP` is copied from CGMES `minP`
+- `MaxP` is copied from CGMES `maxP`
+- `TargetP`/`TargetQ` are set from `SSH` or `SV` values depending on which are defined. CGMES values for `p`/`q` are given with load sign convention, so a change in sign is applied when copying them to `TargetP`/`TargetQ`. If underfined, they are set to `0`.
```
Comment (colinepiloquet) [inline pages/documentation/grid/formats/cim-cgmes.md:312]: Typo here: undefined

**Review by colinepiloquet — APPROVED** [review]

**Review by So-Fras — APPROVED** [review]

**Pull request merged**